assets/prometheus-k8s/prometheus.yaml

```diff
@@ -39,6 +39,7 @@ spec:
   configMaps:
   - serving-certs-ca-bundle
   - kubelet-serving-ca-bundle
+  - hub-router-ca
```

Comment:
I do not think you should add `hub-router-ca` here. I think you can make the `cluster-monitoring-config` configmap to support configuring configmap so that it can be able to update with customized configmap externally. I am not sure if it is correct approach. @simonpasquier can you give us recommendations? Thanks.